docs/FAQ/installing/installing.md

```diff
@@ -0,0 +1,40 @@
+# Installing
+
+
+**What knowledge do I need to install this?**
+
+Depending on the installation way, less specific skill is necessary. For instance, in installation using Docker just follow the tutorial [here](https://docs.shellhub.io/self-hosted/deploying) and that's it! You have the ShellHub on your device.
+
+**To install the ShellHub agent do I need to have Docker?**
+
+No, the ShellHub agent can also be installed manually as shown [here](https://docs.shellhub.io/developers/agent/installing), or, in the case of the Yocto Project image, in the image generated, it can be included in settings.
+
+**I don’t use the Yocto Project and I work with low-power devices such as Raspberry Pi, Thinker, and others. Is it possible to install the ShellHub agent on them?**  
+
+Yes, in this case, use the [manually-install](https://github.com/shellhub-io/shellhub).
```

Comment:
Most of provided examples also run Docker so manual install wouldn't be needed. Please avoid giving examples. I believe this could be inserted in previous question so we offer Yocto Project and manual installation options.